### PR TITLE
Add CSS examples - part 2

### PR DESCRIPTION
**Pull request by kaevupoiss**
Added CSS examples for:
- context-menu
- dialog/alert
- dialog/drawer
- dialog/main

Mention #148 for tracking purposes.

### CODE REVIEW

File: src/docs/previews/context-menu/main/css.svelte [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+		<div class="rightSlot">⇧⌘N</div>
+	</div>
+	<div melt={$separator} class="separator" />
+	<div class="item" melt={$item}>
+		Quit Melt UI
+		<div class="rightSlot">⌘Q</div>
+	</div>
+</div>
+
+<style lang="postcss">
```
Comment (TGlide) [inline src/docs/previews/context-menu/main/css.svelte:101]: Now that I think about it... We should not use lang="postcss" for these examples. Which I think shouldn't be a hassle to change

Comment (kaevupoiss) [reply src/docs/previews/context-menu/main/css.svelte:101]: There are more peculiarities with all of the CSS examples I would like to fix.

For example, I would like to remove all of the --tw variables from the examples. What do you think?

Comment (TGlide) [reply src/docs/previews/context-menu/main/css.svelte:101]: I think that would be amazing tbh

Comment (kaevupoiss) [reply src/docs/previews/context-menu/main/css.svelte:101]: If that's alright, I would love to do all of those changes in a separate PR.

Comment (shyakadavis) [reply src/docs/previews/context-menu/main/css.svelte:101]: Hi.

Mind if I weigh in...

I was already working on other css examples(the old way) and given this info, guess I should hit pause, right?

Also, @kaevupoiss for someone hoping to build on top those fixes, mind sharing some insight into how you plan to approach it. Was hoping I could help if that's an option.

Comment (TGlide) [reply src/docs/previews/context-menu/main/css.svelte:101]: > If that's alright, I would love to do all of those changes to all of the existing CSS examples in a separate PR.

Awesome! Will merge this PR and leave it to you then 😄

Comment (TGlide) [reply src/docs/previews/context-menu/main/css.svelte:101]: > Hi.
> 
> Mind if I weigh in...
> 
> I was already working on other css examples(the old way) and given this info, guess I should hit pause, right?
> 
> Also, @kaevupoiss for someone hoping to build on top those fixes, mind sharing some insight into how you plan to approach it. Was hoping I could help if that's an option.

Probably, yes! Feel free to coordinate over a GH discussion here, or in the Discord channel.

Comment (kaevupoiss) [reply src/docs/previews/context-menu/main/css.svelte:101]: @shyakadavis Feel free to write me in Discord. @revalbuilding

Comment (kaevupoiss) [reply src/docs/previews/context-menu/main/css.svelte:101]: @shyakadavis Feel free to write me in Discord. @revalbuilding

**Pull request merged**